GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+| [Core developer](#Core-developers) | direct write access, issue/PR management, veto right, voting, nomination | nomination by core developers, vote by core developers, 2/3 majority |
+| [CoC committee member](#CoC-committee-members) | CoC maintenance, investigation and enforcement | nomination by core developers, vote by core developers, 2/3 majority and simple CoC majority | 
+| [CC member](#CC-members) | conflict resolution, technical leadership, project management | nomination by core developers, vote by core developers, 2/3 majority and simple CC majority |
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms – not only code – as detailed in the [contributing guide](https://github.com/alan-turing-institute/sktime/blob/master/CONTRIBUTING.rst). 
+
+For more details on how we acknowledge contributions, see the [Acknowledging contributions](#Acknowledging-contributions) section below.
+
+Former and current contributors are listed in [`CONTRIBUTORS.md`](https://github.com/alan-turing-institute/sktime/blob/master/CONTRIBUTORS.md).
+
+### Algorithm maintainer
+
+Algorithm maintainers are contributors who have contributed an algorithm. They have the same voting rights as core developers with regard to their algorithm. 
+
+In sktime, algorithms are encapsulated in classes with specific interface requirements and are called estimators. 
+
+Maintainership operates on the level of classes. To faciliate maintainership questions, we try to write algorithms in separate files when possible. If an algorithm uses multiple files, we try to collect them in a folder.
+
+#### Rights and responsibilities
+
+| Right/responsibility | Description | 
+|---|---|
+| Decision making with respect to their algorithm  | Algorithm maintainers can partipate in the decision making process by vetoing changes and casting votes with regard to proposed changes to their algorithm. This does not extend to proposed changes to the common framework and API. | 
+| Maintenance | They are responsible for maintaining the code and documentation for their algorithm, including bug fixes, unit testing, coding style, compliance with the common API, docstrings, documentation and tutorials notebooks. | 
+| Support | They are the first point of contact for users and other contributors for all questions, issues and proposals regarding their algorithm. |
+
+#### Appointment
+
+The contributor who contributes an algorithm is automatically appointed as its first maintainer. If they can no longer fulfil their maintenance responsibilities, maintainers are expected to resign. 
+
+The maintainer can appoint another contributor as maintainer. No vote is required. An algorithm can have multiple maintainers. 
```

Comment:
there might be a loophole here: by virtue of this rule, any algorithm maintainer can appoint 1000 others. That would give them, by the rule in line 45, the possibility to outvote the core team by 1001 to something.